### ISSUE TITLE
Code colour formatting not working correctly

### ISSUE THREAD

**Issue by chaoscreater** [Gisto/Gisto#285]
Hi there,

I'm a sysadmin and work with mostly powershell. In Gisto, I've configured the settings to use powershell by default. When I create a new snippet, this is what I see:

![image](https://user-images.githubusercontent.com/18227319/81867960-9fc7f880-95c5-11ea-90af-ccc4ba6bbc30.png)

That's great, but when I click save, the colour formatting is incorrect. 

![image](https://user-images.githubusercontent.com/18227319/81868112-e74e8480-95c5-11ea-96c3-a76a1ac618c4.png)

I have to change the filename to specifically .ps1 in order for the colour formatting to apply:

![image](https://user-images.githubusercontent.com/18227319/81868092-dc93ef80-95c5-11ea-95b1-a5db54abad98.png)

**Comment by sanusart** [comment]
Yes, I know it's annoying, I was thinking about it a lot, but this mapping of file extensions - I cannot get it from anywhere. Manually mapping them all will take a lot of time and will not cover all the cases. This happens to all the files types. I guess I need to start mapping it. At least for a few.

**Comment by chaoscreater** [comment]
I have an idea. How about in the settings where you set the default language, you add a field next to it to allow users to specify the extension themselves? So in my case, it'll look something like this:

`Powershell 		.ps1
`

You offload this mapping process to the users themselves. We only have to do this once and we only need to do it for the languages that we work with.
